src/rdb.c

```diff
@@ -1080,6 +1083,10 @@ int rdbSaveBackground(char *filename, rdbSaveInfo *rsi) {
         /* Child */
         closeListeningSockets(0);
         redisSetProcTitle("redis-rdb-bgsave");
+        if (server.bgsave_cpu_set != NULL) {
+          // TODO(caleb): Seems racy. Make a copy before forking or something?
```

Comment:
Isn't Linux fork guaranteed to be copy-on-write?

Reply:
Right.